packages/salesforcedx-apex-debugger/package.json

```diff
@@ -52,8 +52,8 @@
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "test": "npm run test:unit && npm run test:integration",
-    "test:unit": "jest --coverage",
-    "test:integration": "jest --coverage -c ./jest.integration.config.js"
+    "test:unit": "jest --coverage --forceExit",
+    "test:integration": "jest --coverage --forceExit -c ./jest.integration.config.js"
```

Comment:
These tests alway complain about open handles and in GHA hang instead of exiting like they do in circle so I added the forceExit flag